### DISCUSSION TITLE
What country are Revolts servers in and how is data there currently stored?

### DISCUSSION THREAD

**Discussion by hardtofin** [stoatchat/discussions#644] [General]
As there is currently no E2EE in reovlt, i was wondering if you can tell me where (which country) the servers are currently hosted please and by which company i.e where is the company based.  Also is server data just held in plain text format there?

Finally do you have any timeline for when E2EE may be implemented, i see it has been on the roadmap already for a couple of years.  

Many thanks for any help you can give.

**Comment by Rexogamer** [comment]
- Most of our data is stored in Finland by Hetzner; however, files are currently hosted in Lithuania by a member of the team.
- I'm not personally sure about how it's stored so cc @insertish 
- No timeline for E2EE at the moment; we're currently working on a new version of the web app alongside mobile apps for iOS and Android.

**Reply by hardtofin** [reply]
Thanks for this info.  When i look at open connections that revolt makes i only see 2 connections which are both in amsterdam.  Nothing in lithuania or finland?

The reason i am asking all these questions is i need a secure messaging app and so am weighing up the pros and cons.  Your interface and UX is fantastic, but i need to know about the security side.

Are you able to shed any light on the above questions please?

Many thanks.

**Reply by insertish** [reply]
All of our traffic is routed through Cloudflare.

The 'rough timeline' for e2ee is after the new client + voice/video is released.
Although I'm currently thinking planning for e2ee should start immediately after the new client is ready.

**Comment by hardtofin** [comment]
Thanks so much for the personal response, this is great.  I know a large portion of the crypto community would be interested in using revolt as the current default is discord which is obviously a privacy nightmare.  So the E2EE aspect of revolt will draw huge swatches of that community across.  Do you have a rough time line on when the new client may be ready?

Thanks again for your personal response.
